src/main/java/frc/robot/commands/MoveToHome.java

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
```

Comment:
Ew comments (im nitpicking but I think its more readable without these). Otherwise this command looks great.